docs/reference/errors-and-warnings/NU3033.md

```diff
@@ -0,0 +1,41 @@
+---
+title: NuGet Warning NU3033
+description: NU3033 Warning code
+author: heng-liu
+ms.author: henli
+ms.date: 3/18/2019
+ms.topic: reference
+ms.reviewer: dtivel
+f1_keywords: 
+  - NU3033
+---
+
+# NuGet Warning NU3033
+
+## Scenario 1
+
+<pre>Package 'SamplePackage v1.0.0' from source 'https://contoso.com/index.json': A repository primary signature must not have a repository countersignature.</pre>
+
+### Issue
+
+The primary signature should be either an author signature or a repository signature. It must not be a repository countersignature. A repository primary signature cannot have a repository countersignature.
```

Comment:
Remove "It must not be a repository countersignature."  It's absurd.  A primary signature **can't** be a countersignature.

Reply:
Thanks! Fixed.